partials/pagebuilder-components/brand-page.php

```diff
@@ -5,12 +5,29 @@
 
 $brand_text = get_sub_field('brand_text');
 
+$herman_sound_text = get_sub_field('sound_text');
+$herman_sound_image = get_sub_field('sound_image');
+$herman_sound_file = get_sub_field('sound_file');
+
 ?>
 
 <section>
   <div class="flex flex-col-reverse md:flex-row gap-10">
-    <div class="px-2.5 pb-8 md:pt-28 sm:basis-2/3 md:basis-2/5 text-gray-900 text-base">
-      <p><?php echo $brand_text; ?></p>
+    <div class="px-3 pb-8 md:pt-28 sm:basis-2/3 md:basis-2/5 text-gray-900 text-base">
+      <script>
+        function play(){
+            var audio = document.getElementById("audio");
+            audio.play();
+                      }
+      </script>
+
+      <div class="flex">
+          <p><?php echo $herman_sound_text; ?></p>
+          <img class="h-[20px] mt-6 pl-2 cursor-pointer" src="<?php echo $herman_sound_image; ?>" value="PLAY"  onclick="play()">
+         <audio id="audio" src="<?php echo $herman_sound_file; ?>" ></audio>
+      </div>
+      
+      <p class="mt-4"><?php echo $brand_text; ?></p>  
```

Comment:
Zou dit alles nog wrappen in if statement.

zoiets:

if ($herman_sound_text || $herman_sound_image || $herman_sound_file ) {

}

Reply:
Goeie